spec/orm/activerecord_spec.rb

```diff
@@ -1264,6 +1264,14 @@ def monkey
         expect(@event.images_changed?).to be_truthy
         expect(@event.changed_for_autosave?).to be_truthy
       end
+
+      it "should not update image when save with select" do
+        @event[:images] = [stub_file('test.jpeg')]
```

Comment:
Why not `@event.images =` ?

Reply:
I used the old code as a reference.
Fixed `@event.images=` because it was natural.
https://github.com/carrierwaveuploader/carrierwave/pull/2613/files#diff-48bc92466ff9653e1ae5c5b60134a0c7c631b0d414f0d9b0b4642e84097a650bR1277